src/components/QuizTemplate.tsx

```diff
@@ -84,13 +104,64 @@ const QuizTemplate: React.FC<QuizProps> = QuizProps => {
     setChooseAnswer(false);
     setPoints(0);
     setQuestionNumber(1);
+    resetCategoryPoints();
+  };
+
+  const hideResultsBreakdown = () => {
+    setShow(false);
+  };
+
+  const resetCategoryPoints = () => {
+    setAccessibilityQuestionsCorrect(0);
+    setAgileQuestionsCorrect(0);
+    setCSSQuestionsCorrect(0);
+    setGeneralCSQuestionsCorrect(0);
+    setGitQuestionsCorrect(0);
+    setHTMLQuestionsCorrect(0);
+    setInfoTechQuestionsCorrect(0);
+    setJavascriptQuestionsCorrect(0);
+    setLinuxQuestionsCorrect(0);
+    setPythonQuestionsCorrect(0);
+    setQualityAssuranceQuestionsCorrect(0);
+    setSecurityQuestionsCorrect(0);
+    setSQLQuestionsCorrect(0);
   };
 
   const shuffleModalResponses = (responses: string[]) => {
     const shuffleModalArr = shuffle<string>(responses);
     return shuffleModalArr[0];
   };
 
+  const incrementCategoryQuestionScore = (categoryName: string) => {
+    if (categoryName == CATEGORY_NAMES.AGILE) {
+      setAgileQuestionsCorrect(curr => curr + 1);
+    } else if (categoryName == CATEGORY_NAMES.ACCESSIBILITY) {
+      setAccessibilityQuestionsCorrect(curr => curr + 1);
+    } else if (categoryName == CATEGORY_NAMES.CSS) {
+      setCSSQuestionsCorrect(curr => curr + 1);
+    } else if (categoryName == CATEGORY_NAMES.GENERAL) {
+      setGeneralCSQuestionsCorrect(curr => curr + 1);
+    } else if (categoryName == CATEGORY_NAMES.GIT) {
+      setGitQuestionsCorrect(curr => curr + 1);
+    } else if (categoryName == CATEGORY_NAMES.HTML) {
+      setHTMLQuestionsCorrect(curr => curr + 1);
+    } else if (categoryName == CATEGORY_NAMES.INFOTECH) {
+      setInfoTechQuestionsCorrect(curr => curr + 1);
+    } else if (categoryName == CATEGORY_NAMES.JAVASCRIPT) {
+      setJavascriptQuestionsCorrect(curr => curr + 1);
+    } else if (categoryName == CATEGORY_NAMES.LINUX) {
+      setLinuxQuestionsCorrect(curr => curr + 1);
+    } else if (categoryName == CATEGORY_NAMES.PYTHON) {
+      setPythonQuestionsCorrect(curr => curr + 1);
+    } else if (categoryName == CATEGORY_NAMES.QUALITYASSURANCE) {
+      setQualityAssuranceQuestionsCorrect(curr => curr + 1);
+    } else if (categoryName == CATEGORY_NAMES.SECURITY) {
+      setSecurityQuestionsCorrect(curr => curr + 1);
+    } else if (categoryName == CATEGORY_NAMES.SQL) {
+      setSQLQuestionsCorrect(curr => curr + 1);
+    }
```

Comment:
I think we should explore other options to cut down on the repetitiveness

Reply:
I tried using a copy and adding the values as needed. I can't do an instant lookup because the values aren't 1 to 1.

Reply:
Why doesn't it say it's out of date yet?